nil/core-concepts/transaction-lifecycle.mdx

```diff
@@ -0,0 +1,51 @@
+# Transaction lifecycle
+
+## Definition
+
+In =nil;, a transaction passes the following stages before reaching finality.
+
+1. A transaction is submitted by an account
+2. (Optional) Validators track the transaction to determine the shard where the transaction is supposed to be included in a block
```

Comment:
I believe, it's up to RPC, no the validator

Reply:
I have rephrased this point to reflect the PoC spec